samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -198,13 +198,21 @@ void disconnect_cloud(void)
  */
 static bool connect_cloud(void)
 {
+	char device_id[NRF_CLOUD_CLIENT_ID_MAX_LEN + 1];
+	int err;
+
 	LOG_INF("Connecting to nRF Cloud");
 
+	err = nrf_cloud_client_id_get(device_id, sizeof(device_id));
+	if (!err) {
+		LOG_INF("Device ID: %s", device_id);
+	} else {
+		LOG_ERR("Error requesting the device id: %d", err);
+	}
```

Comment:
I think this should be part of the nrf_cloud library (specifically, a feature of `nrf_cloud_connect`) and enabled by a KConfig, rather than part of the sample

But it's weird, I could have sworn we already had something like this?

Reply:
I'd like to keep it this way for now since the REST samples do it in the sample itself as well.

Let's talk about a consolidating it in the library.